src/bindings/python/tests/test_torchvision_to_ov/test_preprocessor.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
+import copy
+import os
+import pytest
+from PIL import Image
+
+import torch
+import torch.nn.functional as F
+import torchvision.transforms as transforms
+
+from openvino.runtime import Core
+
+from openvino.preprocess.torchvision_to_ov import PreprocessConverter
+
+
```

Comment:
Also, I think it makes sens to add it into PyTorch layer tests scope

Reply:
These tests do not match the definition of layer tests. If there's a better place we can put preprocessing tests I'll move them somewhere else, but I don't think they belong to `openvino/tests/layer_tests`.

Reply:
The problem we have here is that different categories of tests (read as "folder with python code") require the same frameworks. 
And it results to the fact we have requirements files, where we have a lot of frameworks like ONNX and PT. We already have a ticket (and PR) to move ONNX tests from python folder to folder closer to ONNX FE. Similar with PR tests - what are we testing here? Python API ? or integration with PT? I suppose the later one, so here I aligned with @rkazants that such tests are closer to PT tests, which allows to group tests by frameworks:
- (No frameworks) - OpenVINO Python API, which does not require heavy dependencies
- (PT) - PT Layer tests, PT decoder, preprocessing tests
- (ONNX) - ONNX tests
- ...

Alternatively, we need to create tox.ini files where developers can test only Python API or only PT preprocessing or only interaction of ONNX and OpenVINO. Currently, we have a mix of everything in one place - OpenVINO Python API, ONNX tests and newly introduced PT preprocessing, also we have different decoders in openvino.frontends package. And it's hard to test everything together.